src/runtime/pay-client.js

```diff
@@ -372,9 +372,16 @@ export class RedirectVerifierHelper {
     // b. WebCrypto (crypto.subtle);
     // c. Crypto random (crypto.getRandomValues);
     // d. SHA284 (crypto.subtle.digest).
+    let canUseLocalStorage;
+    try {
+      canUseLocalStorage = !!this.win_.localStorage;
```

Comment:
http://chrisberkhout.com/blog/localstorage-errors/

It's also possible it can fail when you attempt to write to it because it's full.

Reply:
Thanks for the heads up!

Reply:
I updated the variable name and added a comment